framework/src/modules/network/index.js

```diff
@@ -0,0 +1,63 @@
+const defaults = require('./defaults');
+const Network = require('./network');
+const BaseModule = require('../base_module');
+
+/* eslint-disable class-methods-use-this */
+
+/**
+ * Network module specification
+ *
+ * @namespace Framework.Modules
+ * @type {module.NetworkModule}
+ */
+module.exports = class NetworkModule extends BaseModule {
+	/* eslint-disable-next-line no-useless-constructor */
+	constructor(options) {
+		super(options);
+	}
+
+	static get alias() {
+		return 'network';
+	}
+
+	static get info() {
+		return {
+			author: 'LiskHQ',
+			version: '0.1.0',
+			name: 'lisk-core-network',
+		};
+	}
+
+	get defaults() {
+		return defaults;
+	}
+
+	get events() {
+		return ['ready'];
+	}
+
+	get actions() {
+		return {
```

Comment:
Is this the standard to follow? Why not defining the actions in the constructor?

Reply:
That's how it's implemented in the chain module so I followed the pattern for consistency.

Reply:
Yes, actions need to be actually defined here, if they're defined in the constructor then they won't be registered and won't be able to be called.